php/libraries/Project.class.inc

```diff
@@ -186,6 +224,34 @@ class Project
         $this->_projectName = $projectName;
     }
 
+    /**
+     * Get this project's alias
+     *
+     * @return string This project's alias
+     */
+    public function getAlias(): string
+    {
+        return $this->_projectAlias;
+    }
+
+    /**
+     * Set this project's alias
+     *
+     * @param string $projectAlias This project's alias
+     *
+     * @return void
+     */
+    public function setAlias(string $projectAlias): void
+    {
+        if (strlen($projectAlias) > 4 ) {
+            throw new ConfigurationException(
+                "The length of the project's alias can not be longer than ".
+                "4 characters."
+            );
+        }
+        $this->_projectAlias = $projectAlias;
```

Comment:
This should also update the Database value, I think.

Reply:
i made it similar to the name and id...

Reply:
Fair enough, I didn't look at the rest of the class.